addons/hr_attendance/views/res_config_view.xml

```diff
@@ -18,7 +18,7 @@
     </record>
 
     <record id="action_hr_attendance_settings" model="ir.actions.act_window">
-        <field name="name">Attendance Settings</field>
```

Comment:
Hello, I came on this commit by accident.

I believe the best solution for hr_attendance would be to inherit base.config.settings and name it hr_attendance.config.settings, not just rename the attendance settings name.

It should have been coded that way from the start but this was my first module as a junior developer and I copied a few wrong things from elsewhere in the code, sorry ;-).